### ISSUE TITLE
Potentially missed observation from Polygon

### ISSUE THREAD

**Issue by fadeev** [zeta-chain/node#3308]
https://polygonscan.com/tx/0x06a0ec87f5c6dab11eed5da1357035590d7d54559679c4b1dc3d07264cb51b30

**Comment by lumtis** [comment]
Found the tx, the validation can't be processed https://zetachain.explorers.guru/transaction/764A5A9207CC844EC3A26300C3FB8C7B35E58E194EF25E5CD8C03FB68E6E9614

On the condition that produces the error is that it seems we're doing a BTC withdraw in the crosschain call that doesn't work because it is less than theBTC dust amount.

On the error itself, it seems that under this workflow, the observation tx fails at the Cosmos level, failing the whole txs making the processing of the inbound impossible

**Comment by nzmpi** [comment]
@lumtis That's my tx. Thank you for you answer!
Does this mean that if I am trying to send less BTC than dust amount (1000?) the transaction will revert, and without calling `OnRevert` on the original EVM chain? So those funds are just lost?

**Comment by lumtis** [comment]
No, it is not lost, it is blocked until there is a fix applied.
Did you make other deposits that didn't get observed?

**Comment by nzmpi** [comment]
> Did you make other deposits that didn't get observed?

Not yet, do I need to update my contract on Zetachain to prevent this in future? Like reverting if a user tries to send too little BTC?

**Comment by lumtis** [comment]
We need to upgrade the protocol to fix the workflow and make the CCTX revert, once implemented this currently blocked CCTX will get observed and reverted.
But before upgrade, yes, you can add a check in the smart contract. The Bitcoin dust amount is 1000sat, if you add a `require(withdrawAmount > 1000)` then the CCTX should be reverted. This check will be no longer necessary after Bitcoin upgrade.
